src/main/resources/liquibase/harness/change/expectedSql/mssql/mergeColumns.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE full_name_table (first_name varchar(50), last_name varchar(50))
+INSERT INTO full_name_table (first_name) VALUES ('John')
+UPDATE full_name_table SET last_name = 'Doe' WHERE first_name='John'
+INSERT INTO full_name_table (first_name) VALUES ('Jane')
+UPDATE full_name_table SET last_name = 'Doe' WHERE first_name='Jane'
+ALTER TABLE full_name_table ADD full_name varchar(255)
+UPDATE full_name_table SET full_name = first_name + ' ' + last_name
+DECLARE @sql [nvarchar](MAX)
```

Comment:
@AlexanderSashchenko have you checked data in column after this query for MSSQL? It reminds me about https://github.com/liquibase/liquibase/issues/1818

Reply:
Yes I did. The data is consistent and what it is expected to be. It is just about this strange query for dropping the column. There is no need to create another issue as it is all about dropColumn changeType.